module-2/lab-subsetting-and-descriptive-stats/your-code/main.ipynb

```diff
@@ -194,14 +194,33 @@
   },
```

Comment:
- One way for this question: Find the mean and the standard deviation of the temperature of each state.
Hint: Use functions from Data Manipulation lesson.
```Python
temp.groupby("State").agg({'Temperature':['mean','std']})
```
- You have missed to comment the histograms and the boxplots.
- Why do you use twice ?
```Python 
stats['IQR'] = stats['75%'] - stats['25%'] 
```
- It's a good practice to skip useless columns when importing a data set. This is the case of the unnamed column in challenge 3. This way you avoid misunderstandings with the index.